src/test/java/org/gridsuite/directory/notification/server/DirectoryNotificationWebSocketHandlerTest.java

```diff
@@ -47,6 +46,7 @@ public class DirectoryNotificationWebSocketHandlerTest {
 
     private ObjectMapper objectMapper;
     private WebSocketSession ws;
+    private WebSocketSession ws2;
```

Comment:
?

Reply:
to mock it differently from the other as i don't have the same needs